cgmes/cgmes-extensions/src/main/resources/xsd/cgmesIidmMapping.xsd

```diff
@@ -14,12 +14,18 @@
         <xs:attribute name="side" use="required" type="xs:int"/>
         <xs:attribute name="topologicalNodeIds" use="required" type="ci:nonEmptyString"/>
     </xs:complexType>
+    <xs:complexType name="Base">
```

Comment:
Maybe rename it `BaseVoltage`? (same for name below)

Reply:
Done